src/main/java/uk/gov/companieshouse/alphabeticalcompanysearchconsumer/exception/UpsertServiceException.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.companieshouse.alphabeticalcompanysearchconsumer.exception;
+
+public class UpsertServiceException extends RuntimeException  {
```

Comment:
can't see where this is used. Is this required?

Reply:
Used once in the upsertService method in the UpsertService class.